packaging/torcharrow/build.sh

```diff
@@ -0,0 +1,12 @@
+#!/usr/bin/env bash
+# Copyright (c) Meta Platforms, Inc. and affiliates.
+# All rights reserved.
+#
+# This source code is licensed under the BSD-style license found in the
+# LICENSE file in the root directory of this source tree.
+
+set -ex
+
+git submodule update --init --recursive
+scripts/build_mac_dep.sh ranges_v3 fmt double_conversion folly re2
```

Comment:
Is this intentional to only build on mac?

Reply:
actually on my local copy I often comment out this line when running the build, because I already have the dependencies installed and there is a `prompt` command in `build_mac_dep.sh` that doesn't run very well with the conda build process.

Reply:
I thought conda-build would create a new isolated environment and you have to install all deps within this env.

Reply:
yep, so this script should work without running part with `prompt` if it's ran on a fresh build environment. I'll update the PR with this line removed and later add it to the github action workflow.

Reply:
Cool. LMK when you have gha created. Then, we can see which part is missing.